config/src/test/java/beanconfig/ObjectsConfig.java

```diff
@@ -24,6 +26,14 @@ public String getOptionalValue() {
         public void setOptionalValue(String optionalValue) {
           this.optionalValue = optionalValue;
         }
+
+        public String getNullableValue() {
+            return nullableValue;
+        }
+
+        public void setNullableValue(String nullableValue) {
+            this.nullableValue = nullableValue;
```

Comment:
Though i can see the argument for overwriting it with null always (always ignore the bean constructor) maybe ?